Stanford/JumpstartEngineering/Install/CAPx/CAPxDisplay.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * @file
+ * Define the context for the CAP display view mode for person pages.
+ */
+
+namespace Stanford\JumpstartEngineering\Install\CAPx;
+/**
+ *
+ */
+class CAPxDisplay extends \AbstractInstallTask {
```

Comment:
Yeah, I think that side of me is winning as well.

I have a ticket to create a tool that would help support listing out all off the dependencies to the console/terminal. Lets start putting the dependencies in with the assumption we are going to have this tool available in the shortish future. Otherwise grepping for keywords is what it is going to have to be.